modules/gateways/razorpay/rzpordermapping.php

```diff
@@ -0,0 +1,73 @@
+<?php
+use Illuminate\Database\Capsule\Manager as Capsule;
+
+class RZPOrderMapping
+{
+    private $name;
+
+    function __construct($name)
+    {
+        $this->name = $name;
+    }
+
+    function createTable()
+    {
+        if (!Capsule::schema()->hasTable('tblrzpordermapping'))
+        {
+            Capsule::schema()->create('tblrzpordermapping', function($table) {
+                $table->increments('id');
+                $table->string('merchant_order_id', 30);
+                $table->string('razorpay_order_id', 50);
```

Comment:
@rohitcbr , for RZP order id lets just have fix length for columns as well.

Reply:
updated it here https://github.com/razorpay/razorpay-whmcs/blob/5a11e2c69b505809069c3903bc58d89c216a5e85/modules/gateways/razorpay/rzpordermapping.php#L19